src/fill-zulipdb-with-msgs-deploy.mjs

```diff
@@ -0,0 +1,192 @@
+// Reference to TypeScript definitions for IntelliSense in VSCode
+/// <reference path="../rnode-grpc-gen/js/rnode-grpc-js.d.ts" />
+
+import postgres from 'postgres'; 
+import grpcLib from '@grpc/grpc-js';
+import { rhoParToJson } from '@tgrospic/rnode-grpc-js';
+import util from 'util';
+
+// RNode with environment parameters
+import { rnodeService } from './rnode-env.mjs';
+
+// Load .env file
+import { config } from 'dotenv';
+config();
+
+/**
+  * @param {typeof process.env} env
+  * @param {typeof postgres} postgres
+  * @param {typeof grpcLib} arg.grpcLib Library '@grpc/grpc-js'
+  */
+
+async function main(env, {postgres, grpcLib}) {
```

Comment:
To designate _complex type_ parameter in TypeScript comment, it must be defined as `{Object}` first and latter use as a prefix.
https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#typedef-callback-and-param
```ts
/**
  * @param {typeof process.env} env
  * @param {Object} arg
  * @param {typeof postgres} arg.postgres
  * @param {typeof grpcLib} arg.grpcLib Library '@grpc/grpc-js'
  */
```